packages/workbox-sw/index.mjs

```diff
@@ -89,20 +97,56 @@ class WorkboxSW {
    * @param {string} moduleName
    */
   loadModule(moduleName) {
-    if (this._options.disableModuleImports) {
-      // This can't be a WorkboxError as we can't rely on workbox-core being
-      // loaded.
-      throw new Error(
-        `Attempted to load '${moduleName}' while disableModuleImports is true.`
-      );
+    if (!this._options.disableModuleImports) {
+      let modulePath = this._getImportPath(moduleName);
+      try {
+        importScripts(modulePath);
+      } catch (err) {
+        // We can't rely on workbox-core being loaded so using console
+        console.error(
+          `Unable to import module '${moduleName}' from '${modulePath}'.`);
```

Comment:
Maybe add a `TODO` to provide some more context about this error. Like, if it's a scenario where we know they're using local scripts rather than a CDN copy, then this failure might be due to only deploying `workbox-sw.js` and not the helper modules. Or if it's a CDN path that fails, it's probably due to just using a bad `moduleName`.